codeviz.py

```diff
@@ -146,16 +146,19 @@ def create_dot_file(nodes, edges):
         f.write('    node [shape=Mrecord, fontsize=12]\n\n')
         for n in nodes:
             if not args.no_color:
-                if n.filetype == 'source':
-                    if n.highlight:
-                        f.write('    node [fillcolor="#ff9999", style=filled]')
-                    else:
-                        f.write('    node [fillcolor="#ff9999", style=filled]')
-                elif n.filetype == 'header':
-                    if n.highlight:
-                        f.write('    node [fillcolor="#ccffcc", style=filled]')
-                    else:
-                        f.write('    node [fillcolor="#ccccff", style=filled]')
+                try:
+                    if n.filetype == 'source':
+                        if n.highlight:
+                            f.write('    node [fillcolor="#ff9999", style=filled]')
+                        else:
+                            f.write('    node [fillcolor="#ff9999", style=filled]')
+                    elif n.filetype == 'header':
+                        if n.highlight:
+                            f.write('    node [fillcolor="#ccffcc", style=filled]')
+                        else:
+                            f.write('    node [fillcolor="#ccccff", style=filled]')
+                except:
```

Comment:
What's failing here exactly? It'd be great to catch the exact exception, instead of a generic `except` case.